bag/bag/settings/settings.py

```diff
@@ -194,6 +194,10 @@ def _get_docker_host():
 # Security
 
 JWT_SECRET_KEY = os.getenv('JWT_SHARED_SECRET_KEY')
+
+if not JWT_SECRET_KEY:
+    raise ValueError('JWT_SHARED_SECRET_KEY not set')
```

Comment:
FYI: ik heb (buiten dit PR om) de exception op `JWT_SECRET_KEY` weggehaald; deze check gebeurt nu (netjes) in de middleware